### PR TITLE
Add option to configure automatic closing of dialog after action.run()

### PR DESCRIPTION
**Pull request by asoltysik**

### CODE REVIEW

**Comment by asoltysik** [comment]
Done.

File: src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java [hunk 1]
```diff
@@ -35,6 +35,16 @@
             TerminalSize actionListPreferredSize,
             boolean canCancel,
             List<Runnable> actions) {
+        this(title, description, actionListPreferredSize, canCancel, true, actions);
```
Comment (mabe02) [inline src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java]: No need to make a constructor override, just amend the existing one

**Comment by mabe02** [comment]
Like with the other one, if you could rebase to release/3.0, I'll merge it

File: src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java (hunk 1 above)
Comment (avl42) [reply src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java]: I'm a bit confused.  Is compatibility a non-issue?

**Comment by asoltysik** [comment]
@mabe02 should I create a pull request against `master` or `release/3.0` with the rebased code?

File: src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java (hunk 1 above)
Comment (mabe02) [reply src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java]: Package-private, so I'm ok with changing it (as long as lanterna compiles)

**Comment by mabe02** [comment]
release/3.0 please

File: src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java (hunk 1 above)
Comment (avl42) [reply src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java]: Oh, right, I missed that.  Sorry for my noise.

**Pull request merged**